integration-tests/actions/vrfv2_actions/vrfv2_steps.go

```diff
@@ -103,6 +105,46 @@ func DeployVRFV2Consumers(contractDeployer contracts.ContractDeployer, coordinat
 	return consumers, nil
 }
 
+func DeployVRFV2WrapperConsumers(contractDeployer contracts.ContractDeployer, linkTokenAddress string, vrfV2Wrapper contracts.VRFV2Wrapper, consumerContractsAmount int) ([]contracts.VRFv2WrapperLoadTestConsumer, error) {
+	var consumers []contracts.VRFv2WrapperLoadTestConsumer
+	for i := 1; i <= consumerContractsAmount; i++ {
+		loadTestConsumer, err := contractDeployer.DeployVRFV2WrapperLoadTestConsumer(linkTokenAddress, vrfV2Wrapper.Address())
+		if err != nil {
+			return nil, errors.Wrap(err, ErrAdvancedConsumer)
```

Comment:
we should use go's standard package for dealing with errors instead of _"cosmossdk.io/errors"_ package

`fmt.Errorf("%s: %w", "msg", err)`

e.g. [link](https://github.com/smartcontractkit/chainlink/blob/develop/integration-tests/actions/vrfv2_actions/vrfv2_steps.go#L69)